ckanext/datavic_iar_theme/templates/package/snippets/resources_list.html

```diff
@@ -0,0 +1,40 @@
+{# 
+  Renders a list of resources with icons and view links.
+  
+  resources - A list of resources to render
+  pkg - A package object that the resources belong to.
+  
+  Example:
+  
+    {% snippet "package/snippets/resources_list.html", pkg=pkg, resources=pkg.resources %}
+  
+  #}
+  <section id="dataset-resources" class="resources">
+    {% if historical %}
+      <h2>{{ _('Historical Data and Resources') }}</h2>
+    {% else %}
+      <h2>{{ _('Data and Resources') }}</h2>
+    {% endif %}
+  
+    {% block resource_list %}
+      {% if resources %}
+        <ul class="{% block resource_list_class %}resource-list{% endblock %}">
+          {% block resource_list_inner %}
+            {% set can_edit = h.check_access('package_update', {'id':pkg.id }) %}
+            {% for resource in resources %}
+              {% snippet 'package/snippets/resource_item.html', pkg=pkg, res=resource, can_edit=can_edit, historical=historical %}
```

Comment:
Do you really need this `historical` flag? I looks, like using `super` can reduce amount of code here, unless you have reasons not to use it